src/common/apiBase.ts

```diff
@@ -175,7 +175,11 @@ export class ApiBase {
 
         // set header parameters
         requestOptions.headers.Authorization = this.apiKey;
-        requestOptions.headers["User-Agent"] = userAgent;
+        // only override in node
+        if (typeof window === "undefined") {
```

Comment:
As I would imagine we might have other similar switch based on whether we are running within a browser or within node, I would create a separate method like `isRunningOnBrowser()`